tcms/tests/__init__.py

```diff
@@ -227,3 +230,47 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class GetListOffAllPossiblePermissions(BaseCommand):
+
+    # Helper class for getting all possible permissions
+    # Taken from:
+    # timonweb.com/posts/how-to-get-a-list-of-all-user-permissions-available-in-django-based-project/
```

Comment:
Do not copy & paste code from the web. Aside from the obvious reasons you have no idea what the licensing/permission of this piece of code is and whether or not it can be distributed under GPLv2 (which is what we use).

See file `tcms/utils/permissions.py` for pointers!

You will have to create 2 methods in the test class which:
- one that assigns all permissions except the one we check for
- another function which removes all perms and adds the 1 we check for explicitly

Reply:
FYI `BaseCommand` classes are for actions invoked from `manage.py`. Something like this will be used to resolve #1137 but not for tests.